dotCMS/src/main/webapp/WEB-INF/velocity/preview_mode_menu.vtl

```diff
@@ -27,10 +27,10 @@
 #set($wfSchemes = $workflowtool.findSchemesForStruct($dotPageContent.structure))
 #set($wfScheme = '')
 
-#set($wfStep = $workflowtool.findStepByContentlet($dotPageContent.contentObject))
+#set($wfSteps = $workflowtool.findStepsByContentlet($dotPageContent.contentObject))

```

Comment:
This method needs to change at all levels (tool, api, factory) to be singular (findStepByContentlet), since the contentlet can be in one step at a time. Then we need a new method to get the available steps for a Contentlet, (findAvailableStepsForContent). This way be divide the mixed responsibility that "findStep*s*byContentlet currently has.

Reply:
Hey @dsilvam this method will return a list of steps not just one, if the contentet is not in a particular workflow. that's why the method name is not on singular